src/pycolmap/scene/reconstruction.cc

```diff
@@ -252,19 +252,16 @@ void BindReconstruction(py::module& m) {
            })
       .def("__repr__",
            [](const Reconstruction& self) {
-             std::stringstream ss;
-             ss << "Reconstruction(num_reg_images=" << self.NumRegImages()
-                << ", num_cameras=" << self.NumCameras()
-                << ", num_points3D=" << self.NumPoints3D()
-                << ", num_observations=" << self.ComputeNumObservations()
-                << ")";
+             std::ostringstream ss;
+             ss << self;
              return ss.str();
            })
```

Comment:
I suggest merging https://github.com/colmap/colmap/pull/2855 first and then changing this to:
`.def("__repr__", &CreateRepresentation<Reconstruction>)`